Bio/PDB/MMTF/__init__.py

```diff
@@ -0,0 +1,24 @@
+from mmtf import fetch,parse
```

Comment:
There should be a space after the comma for PEP8 conformance (see also 

However, I would replace this with something like this:

```
try:
   from mmtf import fetch, parse
except ImportError:
   from Bio import MissingPythonDependencyError
   raise MissingPythonDependencyError("Install mmtf to use Bio.PDB.mmtf "
                                      "(e.g. pip install python-mmtf)")
```

This serves two goals - first some user guidance when the import fails (this new exception subclasses ``ImportError`` so is still easy to catch), and second it will let ``run_tests.py`` skip this smoothly if the dependency is missing.